buster/chatbot.py

```diff
@@ -32,7 +32,7 @@ class ChatbotConfig:
     embedding_model: OpenAI model to use to get embeddings.
     top_k: Max number of documents to retrieve, ordered by cosine similarity
     thresh: threshold for cosine similarity to be considered
-    max_chars: maximum number of characters the retrieved documents can be. Will truncate otherwise.
+    max_words: maximum number of words the retrieved documents can be. Will truncate otherwise.
```

Comment:
Is it words or token ?

Reply:
right now it's implemented as words, with the rough guesstimate that 4000 tokens limit ~= 3000 words. We have an open issue to start converting everything to proper token counts, shouldn't be too hard just not in the scope of this PR